packages/block-library/src/navigation/edit/index.js

```diff
@@ -710,6 +738,20 @@ function Navigation( {
 					classNames={ overlayClassnames }
 					styles={ overlayStyles }
 				>
+					<BlockControls>
+						<ToolbarGroup className="wp-block-navigation__toolbar-menu-selector">
+							<NavigationMenuSelector
+								ref={ null }
+								currentMenuId={ null }
```

Comment:
Can we explain why these are set to `null`? I assume because we don't have a ref to a post at the moment?

Reply:
This control will go away anyway to the block inspector so the code will go away eventually too. It is here only as an escape hatch for now.